package.json

```diff
@@ -34,6 +34,7 @@
     "preinstall": "node scripts/preinstall.mjs",
     "build": "ember build --environment=production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix:local-dic": "node scripts/lint:fix:local-dic.mjs",
```

Comment:
I decided to be specific, instead of just `lint:fix`, in case we want in the future to implement a "catch-all" script named `lint:fix` that would run all `lint:fix:*` scripts.

Also because, in `CONTRIBUTING.md` we tell users to run `npm run lint` to spellcheck their page, and if we also tell them to run `npm run lint:fix` to sort `.local.dic` that could be confusing as they may expect `lint:fix` to fix the warning of `npm run lint`.